chaudloader/src/hooks/stage0.rs

```diff
@@ -167,6 +167,23 @@ fn init(
             log::error!("[mod: {}] failed to init: {}", mod_name, e);
         }
     }
+    let mut on_game_load_hook_needed = false;
+    {
+        assert!(MODFUNCTIONS
+            .set(std::sync::Mutex::new(ModFunctions::new()))
+            .is_ok());
+        let mut mod_funcs = MODFUNCTIONS.get().unwrap().lock().unwrap();
+        for mod_state in loaded_mods.values() {
+            for dll in mod_state.dlls.values() {
+                unsafe{
+                    if let Ok(on_game_load_symbol_address) = dll.get_symbol_address("on_game_load") {
+                        mod_funcs.on_game_load_functions.push(std::mem::transmute::<winapi::shared::minwindef::FARPROC, fn(u32, *const u8)>(on_game_load_symbol_address));
+                        on_game_load_hook_needed = true;
+                    }
+                }
+            }
+        }
+    }
```

Comment:
Chaudloader currently doesn't need to write anything to the actual exe's sections. I think the concern is if Capcom adds any new antitamper methods for the exe sections in the future it would stop chaudloader from working even for people that are just using font mods or translations which don't need this hook.